src/results/tests.ts

```diff
@@ -2686,7 +2686,7 @@ export const valueTypeSpecializationTests: Test[] = [
       },
     ],
     values: [[':MyType', 1n, 'abc']],
-    displayValues: ['(:MyType, 1, abc)'],
+    displayValues: ['(:MyType, 1, "abc")'],
```

Comment:
Are there any tests for displaying strings with escaped quotes and escaped `\`s inside? I expected to see those getting modified in this PR as well.

Reply:
There weren't any. I'll add one.

Reply:
Thank you.